README.md

```diff
@@ -27,7 +30,7 @@ Please refer to the [installation README](./install/README.md) file for detailed
 ### Run locally
 
 Running the console locally requires the use of a remote or locally-running Kubernetes cluster that hosts the Strimzi Kafka operator
-and any Apache Kafka® clusters that will be accessed from the console. To get started, you will need to provide a console configuration
+and any Apache Kafka<sup>®</sup> clusters that will be accessed from the console. To get started, you will need to provide a console configuration
```

Comment:
(non-blocking) I'd provide the minimal step required to deploy Strimzi in a suitable way.  We want the bar for get started with a contribution to be as low as we can.